File: app/src/main/java/com/gdi/hahi/callback/LiveVideoAllCallBack.java
package com.gdi.hahi.callback;

import android.util.Log;

import com.shuyu.gsyvideoplayer.listener.StandardVideoAllCallBack;
import com.shuyu.gsyvideoplayer.listener.VideoAllCallBack;

/**
 * Created by gandi on 2017/9/10 0010.
 */

public abstract class LiveVideoAllCallBack implements StandardVideoAllCallBack {
    private static final String TAG = "LiveVideoAllCallBack";
    @Override
    public void onPrepared(String url, Object... objects) {
        Log.i(TAG, "onPrepared: ");
    }

    @Override
    public void onClickStartIcon(String url, Object... objects) {
        Log.i(TAG, "onClickStartIcon: ");
    }

    @Override
    public void onClickStartError(String url, Object... objects) {
        Log.i(TAG, "onClickStartError: ");
    }

    @Override
    public void onClickStop(String url, Object... objects) {
        Log.i(TAG, "onClickStop: ");
    }

    @Override
    public void onClickStopFullscreen(String url, Object... objects) {
        Log.i(TAG, "onClickStopFullscreen: ");
    }

    @Override
    public void onClickResume(String url, Object... objects) {
        Log.i(TAG, "onClickResume: ");
    }

    @Override
    public void onClickResumeFullscreen(String url, Object... objects) {
        Log.i(TAG, "onClickResumeFullscreen: ");
    }

    @Override
    public void onClickSeekbar(String url, Object... objects) {
        Log.i(TAG, "onClickSeekbar: ");
    }

    @Override
    public void onClickSeekbarFullscreen(String url, Object... objects) {
        Log.i(TAG, "onClickSeekbarFullscreen: ");
    }

    @Override
    public void onAutoComplete(String url, Object... objects) {
        Log.i(TAG, "onAutoComplete: ");
    }

    @Override
    public void onEnterFullscreen(String url, Object... objects) {
        Log.i(TAG, "onEnterFullscreen: ");
    }

    @Override
    public void onQuitFullscreen(String url, Object... objects) {
        Log.i(TAG, "onQuitFullscreen: ");
    }

    @Override
    public void onQuitSmallWidget(String url, Object... objects) {
        Log.i(TAG, "onQuitSmallWidget: ");
    }

    @Override
    public void onEnterSmallWidget(String url, Object... objects) {
        Log.i(TAG, "onEnterSmallWidget: ");
    }

    @Override
    public void onTouchScreenSeekVolume(String url, Object... objects) {
        Log.i(TAG, "onTouchScreenSeekVolume: ");
    }

    @Override
    public void onTouchScreenSeekPosition(String url, Object... objects) {
        Log.i(TAG, "onTouchScreenSeekPosition: ");
    }

    @Override
    public void onTouchScreenSeekLight(String url, Object... objects) {
        Log.i(TAG, "onTouchScreenSeekLight: ");
    }

    @Override
    public void onPlayError(String url, Object... objects) {
        Log.i(TAG, "onPlayError: ");
    }

    @Override
    public void onClickStartThumb(String url, Object... objects) {
        Log.i(TAG, "onClickStartThumb: ");
    }

    @Override
    public void onClickBlank(String url, Object... objects) {
        Log.i(TAG, "onClickBlank: ");
    }

    @Override
    public void onClickBlankFullscreen(String url, Object... objects) {
        Log.i(TAG, "onClickBlankFullscreen: ");
    }
}

File: app/src/main/java/com/gdi/hahi/callback/LiveVideoAllCallBack.kt
package com.gdi.hahi.callback

import android.util.Log

import com.shuyu.gsyvideoplayer.listener.StandardVideoAllCallBack
import com.shuyu.gsyvideoplayer.listener.VideoAllCallBack

/**
 * Created by gandi on 2017/9/10 0010.
 */

abstract class LiveVideoAllCallBack : StandardVideoAllCallBack {
    override fun onPrepared(url: String, vararg objects: Any) {
        Log.i(TAG, "onPrepared: ")
    }

    override fun onClickStartIcon(url: String, vararg objects: Any) {
        Log.i(TAG, "onClickStartIcon: ")
    }

    override fun onClickStartError(url: String, vararg objects: Any) {
        Log.i(TAG, "onClickStartError: ")
    }

    override fun onClickStop(url: String, vararg objects: Any) {
        Log.i(TAG, "onClickStop: ")
    }

    override fun onClickStopFullscreen(url: String, vararg objects: Any) {
        Log.i(TAG, "onClickStopFullscreen: ")
    }

    override fun onClickResume(url: String, vararg objects: Any) {
        Log.i(TAG, "onClickResume: ")
    }

    override fun onClickResumeFullscreen(url: String, vararg objects: Any) {
        Log.i(TAG, "onClickResumeFullscreen: ")
    }

    override fun onClickSeekbar(url: String, vararg objects: Any) {
        Log.i(TAG, "onClickSeekbar: ")
    }

    override fun onClickSeekbarFullscreen(url: String, vararg objects: Any) {
        Log.i(TAG, "onClickSeekbarFullscreen: ")
    }

    override fun onAutoComplete(url: String, vararg objects: Any) {
        Log.i(TAG, "onAutoComplete: ")
    }

    override fun onEnterFullscreen(url: String, vararg objects: Any) {
        Log.i(TAG, "onEnterFullscreen: ")
    }

    override fun onQuitFullscreen(url: String, vararg objects: Any) {
        Log.i(TAG, "onQuitFullscreen: ")
    }

    override fun onQuitSmallWidget(url: String, vararg objects: Any) {
        Log.i(TAG, "onQuitSmallWidget: ")
    }

    override fun onEnterSmallWidget(url: String, vararg objects: Any) {
        Log.i(TAG, "onEnterSmallWidget: ")
    }

    override fun onTouchScreenSeekVolume(url: String, vararg objects: Any) {
        Log.i(TAG, "onTouchScreenSeekVolume: ")
    }

    override fun onTouchScreenSeekPosition(url: String, vararg objects: Any) {
        Log.i(TAG, "onTouchScreenSeekPosition: ")
    }

    override fun onTouchScreenSeekLight(url: String, vararg objects: Any) {
        Log.i(TAG, "onTouchScreenSeekLight: ")
    }

    override fun onPlayError(url: String, vararg objects: Any) {
        Log.i(TAG, "onPlayError: ")
    }

    override fun onClickStartThumb(url: String, vararg objects: Any) {
        Log.i(TAG, "onClickStartThumb: ")
    }

    override fun onClickBlank(url: String, vararg objects: Any) {
        Log.i(TAG, "onClickBlank: ")
    }

    override fun onClickBlankFullscreen(url: String, vararg objects: Any) {
        Log.i(TAG, "onClickBlankFullscreen: ")
    }

    companion object {
        private val TAG = "LiveVideoAllCallBack"
    }
}
